server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -338,6 +338,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion ESQL_FIXED_INDEX_LIKE = def(9_119_0_00);
     public static final TransportVersion LOOKUP_JOIN_CCS = def(9_120_0_00);
     public static final TransportVersion NODE_USAGE_STATS_FOR_THREAD_POOLS_IN_CLUSTER_INFO = def(9_121_0_00);
+    public static final TransportVersion ESQL_ORDINALS_OPERATOR_STATUS = def(9_122_0_00);
```

Comment:
If you are going to backport you should make the backport transport versions and use them in this PR before merging.

Reply:
You could do it in a follow-up PR, but this PR is small enough I'd do it in this one.

Reply:
Versions added!